docs/setup/getting_started.soy

```diff
@@ -153,30 +444,24 @@ project_config(
 )" > res/com/example/activity/BUCK
 </pre>{/literal}
 
-<h2>Create a Keystore</h2>
+<h3>Create a Keystore</h3>
 
-In practice, you will want to be able to test your Android app on a
-physical Android device, which means that it needs to be signed.
-<p>
-We will create app-specific information, such as the key and manifest,
-in its own directory to keep things tidy:
+In practice, you will want to be able to test your Android app on a physical Android device, which means that it needs to be signed. We will create app-specific information, such as the key and manifest, in its own directory to keep things tidy:
 
 <pre>mkdir -p apps/myapp/</pre>
 
-To keep things simple, we will create a self-signed certificate for
-debugging. Unfortunately, this is not a one-liner because there is a
-number of prompts from the <code>keytool</code> command:
+To keep things simple, we will create a self-signed certificate for debugging.
+
+<p><blockquote>Unfortunately, this is not a one-liner because there is a number of prompts from the <code>keytool</code> command.</blockquote></p>
 
 {literal}<pre>
 keytool -genkey -keystore apps/myapp/debug.keystore -alias my_alias \
     -keyalg RSA -keysize 2048 -validity 10000
 </pre>{/literal}
 
-when prompted for a keystore password, just use <code>android</code>{sp}
-(and then type it again to confirm it), and hit Enter to accept the
-default values for name, organizational unit, etc.
+When prompted for a keystore password, just use <code>android</code> (and then type it again to confirm it), and hit <code>Enter</code> to accept the default values for name, organizational unit, etc.
 
-<p>
+<br/>
```

Comment:
Globally, we should avoid using `<br />`.  Instead, wrap each paragraph in `<p></p>`.